src/app/vote/page.tsx

```
@@ -7,56 +7,105 @@ import color from "@/packages/design-system/src/color";
+  let filteredVotes =
+    activeIdx === 0
+      ? votes
+      : votes.filter((vote) => vote.category === categories[activeIdx]);
+
+  filteredVotes = [...filteredVotes].sort((a, b) => {
+    switch (sortStandard) {
+      case "투표 제작일 기준":
+        const dateA =
+          new Date(a.finishedAt).getTime() - 7 * 24 * 60 * 60 * 1000;
+        const dateB =
+          new Date(b.finishedAt).getTime() - 7 * 24 * 60 * 60 * 1000;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)